process/check.go

```diff
@@ -0,0 +1,41 @@
+package process
+
+import elasticIndexer "github.com/ElrondNetwork/elastic-indexer-go"
+
+func checkArguments(arguments *ArgElasticProcessor) error {
+	if arguments == nil {
+		return elasticIndexer.ErrNilElasticProcessorArguments
+	}
+	if arguments.EnabledIndexes == nil {
```

Comment:
check.IfNil or check.IfNilReflect in case of interfaces

Reply:
done.